test/server/cards/08-AS/DeAnimator.spec.js

```diff
@@ -145,5 +145,20 @@ describe('De-Animator', function () {
             expect(this.botBookton.location).toBe('discard');
             expect(this.player1).toHavePrompt('Choose a card to play, discard or use');
         });
+
+        it('should send æmber on destroyed mineralized creatures to the common supply', function () {
```

Comment:
nit: mineralized doesn't exactly mean anything in the game - creatures can have a mineralize counter on them, and if de-animator is out then the creatures with a mineralize counter are artifacts. But if de-animator is not out then they're still creatures.

Might be more accurate to say "destroyed creatures as artifacts" or something similar.

Reply:
Done.